linux-anvil-2018-08/entrypoint_source

```diff
@@ -0,0 +1,3 @@
+# Activate the `base` conda environment.
+. /opt/conda/etc/profile.d/conda.sh
```

Comment:
Ah, you linked it to `/etc/profile.d/conda.sh` in https://github.com/conda-forge/docker-images/pull/70/files#diff-26d09eb353cff5cdc4b8d7f6b9beac55R25 so I guess I don't need to do it here since `entrypoint` runs Bash as a login shell.

Reply:
Yeah, I just did that recently because I was unable to get the `entrypoint_source` solution (as you have here) to work for me locally. No strong feelings if you have a preference.